baremetal/src/Runtime.cc

```diff
@@ -55,16 +59,48 @@ class Connection : public ebbrt::TcpHandler {
 }  // namespace
 
 void ebbrt::runtime::Init() {
-  auto reader = boot_fdt::Get();
-  auto offset = reader.GetNodeOffset("/runtime");
-  auto ip = reader.GetProperty32(offset, "address");
-  auto port = reader.GetProperty16(offset, "port");
-  auto allocation_id = reader.GetProperty16(offset, "allocation_id");
+  auto cmdline = std::string(ebbrt::multiboot::CmdLine());
+  auto host = std::string("host_addr=");
```

Comment:
I assume `auto foo = std::string("bar");` is equivalent to `std::string foo("bar");`, and both more correct than `auto foo = "bar";`